src/Symfony/Component/Lock/Lock.php

```diff
@@ -83,6 +83,11 @@ public function acquire($blocking = false)
             }
 
             if ($this->key->isExpired()) {
+                try {
+                    $this->release();
+                } catch (\Exception $e) {
```

Comment:
Would be nice to add it as previous `$e`.

Reply:
hmm.. I don't think so.. Here the exception throw by `release` is not related to the `LockExpiredException` we want to throw..

Reply:
Imagine having this caught in production and not knowing the cause.